### DISCUSSION TITLE
DietPi-Software | taiga-docker

### DISCUSSION THREAD

**Discussion by jeppenodgaard** [MichaIng/DietPi#5747] [Software Request]
#### Formal software information
- Software title | Taiga
- Short description | Free and open-source project management tool
- Official URL (if available) | https://www.taiga.io/
- GitHub URL (if available) | https://github.com/kaleidos-ventures/taiga-docker
- (Official) install documentation | https://resources.taiga.io/30min-setup/

#### Are there similar/alternative software titles available with DietPi-Software?

No

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- Taiga requires docker and docker-compose. Then a git repository most be cloned and a certain branch checked out. After that `./launch-all.sh` should be run. Then users has to be added via the shell.
- The installation guide is called "30 minute setup" which should be able to decrease with a wrapper script.

_Note: I tried the manual installation on dietpi but it seemed to get stuck at `./launch-all.sh`._ 

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

I have some bash experience. So I will be able to help out a bit.

**Comment by MichaIng** [comment]
Many thanks for your request. Was there some error message it did it just hang at `launch-all.sh`?

**Comment by jeppenodgaard** [comment]
It kind of hang. The docker compose progress just seem to keep running. I also tried running it in tmux overnight and I was not able to connect via ssh in the morning (my device is headless).
In hindsight it might be due to too little space, but not sure..
